src/celeritas/global/ActionManager.cc

```diff
@@ -7,6 +7,9 @@
 //---------------------------------------------------------------------------//
 #include "ActionManager.hh"
 
+#    include <cuda_runtime_api.h>
```

Comment:
Just for cosmetics, remove white space.

Reply:
Thanks, and this is causing the non-cuda builds to fail! Clang-format must not be set up correctly on the machine I committed this from 😅